packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -227,6 +228,10 @@ const DatePickerInput = (props) => {
     delete inputAttributes.id;
   }
 
+  if (inputAttributes && inputAttributes.fieldLabel) {
+    labelString = inputAttributes.fieldLabel;
+  }
+
```

Comment:
These code changes will not work if the consumer passes the prop name as `label` or anything else instead of `fieldLabel`. How will the consumer know that they have to pass fieldLabel always to the inputAttributes?

Reply:
This is added because of there is no field component in dateTimePicker and to handle a reading of single label in both the component require to use it through fieldLabel .